### ISSUE TITLE
Add target waypoint to Rviz dynamic waypoint publisher

### ISSUE THREAD

**Issue by JMoore5353** [rosflight/rosplane#43]
The visualization currently plots all waypoints and numbers them. Sometimes it is hard to know what the target waypoint is. Add something to the visualizer to know what the next waypoint is.

**Comment by JMoore5353** [comment]
The visualization now has an arrow marker pointing to the target waypoint. 

A TODO still left is to plot the orbit path when flying an orbit.

**Comment by iandareid** [comment]
@JMoore5353 I think that having them as separate as possible is better. Another factor that may influence your decision:

- It would be useful to see the orbit that it is following through the fillets as well.
- Whenever the aircraft follows an orbit, /current_path publishes the center of the orbit and the radius.
    - We could just extract this data and only plot the orbit as it is following it. This would include orbit last.
        - This would have the downside of not visualizing the paths before or after they are actively being followed, but it would be simple and give us the behavior we would like.

So in short maybe there is a third option? I think either of the things you propose would be great, but I would suggest that the less decoupled version may be better.

**Comment by JMoore5353** [comment]
That's great, I like that option better. We would only see the orbit / fillet when the aircraft is following that part of the path, but I think that is acceptable.
